core/session.js

```diff
@@ -372,25 +418,27 @@ function RinqSession (
   }
 
   function dispatchNotification (message) {
-    var type, payload
+    var namespace, type, payload
 
+    namespace = message.namespace
     type = message.notificationType
     payload = message.payload()
 
     if (log) {
       logger(
         [
-          '%c%s %s[recv] [noti] %s',
+          '%c%s %s[recv] [noti] %s %s',
           'color: teal',
           notificationSymbol,
           log.prefix,
+          namespace,
           type
         ],
-        [[{type: type, payload: payload}]]
+        [[{namespace: namespace, type: type, payload: payload}]]
```

Comment:
I guess for consistency, we should probably log listens and unlistens more like this. That is, we don't JSON.stringify(), but rather log the values directly, as a "secondary".